app/scripts/components/exploration/components/map/analysis-message-control.tsx

```diff
@@ -78,7 +78,9 @@ const MessageStatusIndicator = styled.div<MessageStatusIndicatorProps>`
     }
   }}
 `;
-const MessageContent = styled.div`
+const MessageContent = styled.div.attrs({
+    'data-testid': 'analysis-message'
+  })`
```

Comment:
this wasn't getting rendered.  looks like you need to spread it on the MessageContent component in line 200:
```
<MessageContent {...props}>
```

Reply:
Are you sure it does not show up in the DOM? 
<img width="406" alt="Screenshot 2024-10-28 at 11 34 51" src="https://github.com/user-attachments/assets/a5410a83-9ff1-4175-b415-f1bd13d4d6c6"> 
It does for me... and I don't want to touch all the places where the `MessageContent` component is being used, if its not really necessary. From the [styled component docs](https://styled-components.com/docs/faqs#when-to-use-attrs) it seems we don't need to spread the props, but please let me know if you come to a different conclusion!

Reply:
I pulled down your latest commits and re-ran things, and I see it now.  I don't know why I couldn't before. 🤷‍♂️